src/bbs2gh/Handlers/MigrateRepoCommandHandler.cs

```diff
@@ -471,7 +476,7 @@ private void ValidateOptions(MigrateRepoCommandArgs args)
 
     private void ValidateDownloadOptions(MigrateRepoCommandArgs args)
     {
-        var sshArgs = new[] { args.SshUser, args.SshPrivateKey };
+        var sshArgs = new[] { args.ArchiveDownloadHost, args.SshUser, args.SshPrivateKey };
```

Comment:
Since this is an optional arg that can belong to both ssh and smb I would just remove it from ssh args. Instead one extra validation can be added:
```c#
if (args.ArchiveDownloadHost.HasValue() && !shouldUseShh && !shouldUseSmb)
{
    throw new OctoshiftCliException("--archive-download-host should be provided either with SSH or SMB download options.");
}
```

Reply:
Good point! I wrote this before we decided to use the same argument for SSH and SMB.